app/extensions/api/parameters.py

```diff
@@ -70,3 +70,15 @@ class PaginationParameters(Parameters):
         description='the field to reverse the sorted results (after paging has been performed), default is False',
         missing=False,
     )
+
+
+class PaginationParametersLatestFirst(PaginationParameters):
```

Comment:
This can be re-used for `app.modules.notifications.parameters.ListAllUnreadNotifications` and arguably for `app.modules.collaborations.parameters.GetCollaborationsParameters` as well (@brmscheiner does reversing the order of listing all collaborations impact the frontend?  This ordering is only used for the global "list everything in Houston" API, which is intended for site admins).

Reply:
should be just `[GET] /api/v1/collaborations/`

Reply:
I'm inclined to make general tools and re-use them whenever possible to remove duplicated code.  If we are making a general `PaginationParametersLatestFirst` class inside the API extension, let's update the two additional places to use it.

Reply:
Ah I see what you mean WRT the ListAllUnreadNotifications. I hadn't realised it was identical and yes collabs in reverse LatestFirst order does make sense. Just testing it now.